src/Steam.package/STOSSUserInfos.class/class/github..st

```diff
@@ -0,0 +1,3 @@
+instance creation
+github: infos
+	^ STOSSGithubUserInfos new infos: infos
```

Comment:
We should check this `STOSSUserInfos` class. Because I don't like that we extend this class with methods for each kind of Login... The problem is: If a user wants to add a new authentication that we didn't think about, he has to edit this class! It would be better to make it more pluggable :)